forwarder/cmd/forwarder/forwarder.go

```diff
@@ -72,35 +72,52 @@ func getLogs(ctx context.Context, storageClient *storage.Client, cursors *cursor
 	return err
 }
 
-func parseLogs(reader io.ReadCloser, blob storage.Blob, piiScrubber logs.Scrubber, logsChannel chan<- *logs.Log) (int64, int64, error) {
+func parseLogs(ctx context.Context, reader io.ReadCloser, blob storage.Blob, piiScrubber logs.Scrubber, logsChannel chan<- *logs.Log) (int64, int64, error) {
 	var processedLogs int64
 
 	var currLog *logs.Log
 	var err error
-	parsedLogsIter, totalBytes, err := logs.Parse(reader, blob, piiScrubber)
-	if err != nil {
-		return 0, 0, fmt.Errorf("error parsing logs: %w", err)
+	parsedLogsIter, totalBytes, parseErr := logs.Parse(reader, blob, piiScrubber)
```

Comment:
Is this intended to have some different behavior? The usual style is to just use `err` as it was